src/sentry/api/urls.py

```diff
@@ -2906,6 +2907,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         SentryAppPublishRequestEndpoint.as_view(),
         name="sentry-api-0-sentry-app-publish-request",
     ),
+    re_path(
+        r"^(?P<sentry_app_id_or_slug>[^\/]+)/requests-v2/$",
```

Comment:
Instead of encoding `requests-v2` in the path, could we instead name this something different? Like `request-logs` or `webhook-responses`?